src/components/private/timeline/Timeline.jsx

```diff
@@ -1,81 +1,64 @@
-import React, { useEffect, useState } from "react";
-import ghori from "../../../assets/images/timer/ghori.png";
-import letters from "../../../assets/images/timer/letters.png";
-import mask from "../../../assets/images/timer/mask.png";
+import React, { useState, useEffect } from "react";
 import "./Timeline.scss";
+import HeaderData from "../../../assets/data/HeaderContent";
+import { Header } from "../../shared";
+import { EventTimeline } from "../../../assets/data/Timeline";
+import Modal from "./Modal";
 
 const Timeline = () => {
-  const calculateTimeLeft = () => {
-    const targetDate = new Date("2024-06-28T18:00:00");
-    const now = new Date();
-    const difference = targetDate - now;
-
-    let timeLeft = {};
-
-    if (difference > 0) {
-      timeLeft = {
-        days: Math.floor(difference / (1000 * 60 * 60 * 24)),
-        hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
-        minutes: Math.floor((difference / 1000 / 60) % 60),
-        seconds: Math.floor((difference / 1000) % 60),
-      };
-    } else {
-      timeLeft = {
-        days: 0,
-        hours: 0,
-        minutes: 0,
-        seconds: 0,
-      };
-    }
-
-    return timeLeft;
-  };
-
-  const [timeLeft, setTimeLeft] = useState(calculateTimeLeft());
```

Comment:
These are important logics for timer section, the timeline section is to appear after the timer section - so make the changes as soon as possible.